gradio/components/chatbot.py

```diff
@@ -491,7 +492,8 @@ def _postprocess_content(
         | TextMessageDict,
     ) -> Union[TextMessage, FileMessage, ComponentMessage, None]:
         if isinstance(chat_message, str):
-            return TextMessage(text=inspect.cleandoc(chat_message))
+            sanitized_html = html.escape(inspect.cleandoc(chat_message))
```

Comment:
Hi @adanny1278 ! The problem with this solution is that it breaks chatbots that properly format the html code with markdown backticks.

<img width="825" height="426" alt="image" src="https://github.com/user-attachments/assets/1bf643ef-cafb-4904-9577-b29cde772c01" />

I think the right solution is to add backticks to the <html> code if they are not present already